pyubx2/ubxtypes_core.py

```diff
@@ -231,40 +233,40 @@
     # These need special handling as MSGIDs alone are not unique;
     # message identity is determined by 'type' attribute in payload
     # ***************************************************************
-    b"\x13\x60\x01": "MGA-ACK-DATA0",
```

Comment:
To provide better readability, I've sorted this whole block to make the IDs be in ascending order.
No other change made here.

Reply:
Hi - appreciate the contribution and will review. Not sure I agree that sorting the block by msg/id necessarily aids readability - the blocks are currently sorted in alphabetical order of message name ("MGA-ACK-DATA0" etc.) and sorting this block differently creates unnecessary churn IMHO. I'd prefer to limit the delta to just the messages you've added or amended and to state explicitly in the PR preamble which specific messages these are.